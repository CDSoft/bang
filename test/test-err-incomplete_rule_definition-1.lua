generator(false)

rule "doit" {
    description = "...",
    --command = "...",
    depfile = "...",
    deps = "...",
    dyndep = "...",
    pool = "...",
    msvc_deps_prefix = "...",
    generator = "...",
    restat = "...",
    rspfile = "...",
    rspfile_content = "...",
}
