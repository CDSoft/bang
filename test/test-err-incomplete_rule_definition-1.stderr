[test/test-err-incomplete_rule_definition-1.lua:3] ERROR: rule doit: expected 'command' attribute
